### PR TITLE
1.0.5; New package

### PR DESCRIPTION
**Pull request by ViridianMelody**
Upstream Repo: https://github.com/pdm-project/pdm-pep517
Jira issue: https://anaconda.atlassian.net/browse/PKG-703

Newly forked from conda-forge. Will verify successful linux-s390x build before merging

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,15 +12,16 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<37]
```
Comment (aniedzielaAnaconda) [inline recipe/meta.yaml:15]: Why do you skip for Python < 3.7?

Comment (ViridianMelody) [reply recipe/meta.yaml:15]: The source repo requires Python >= 3.7.
https://github.com/pdm-project/pdm-pep517/blob/4bb714d52ff37cd80f44eee47bad42ef742e8332/pyproject.toml#L10

**Comment by aniedzielaAnaconda** [comment]
LGTM

**Review by aniedzielaAnaconda — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]
You can safely update it to 1.0.6 that has been released recently

**Pull request merged**